django_redis/client/default.py

```diff
@@ -108,14 +111,14 @@ def connect(self, index=0):
 
     def set(
         self,
-        key,
-        value,
-        timeout=DEFAULT_TIMEOUT,
-        version=None,
-        client=None,
-        nx=False,
-        xx=False,
-    ):
+        key: Any,
+        value: Any,
+        timeout: Optional[float] = DEFAULT_TIMEOUT,
+        version: Optional[int] = None,
+        client: Redis = None,
+        nx: bool = False,
+        xx: bool = False,
+    ) -> Union[Optional[int], bool]:
```

Comment:
When does this function return `int`? Or `None` for that matter.

Reply:
the delete should return an `int`, actually it returns directly what the client returns, that should be `bytes`